frontend/src/app/add-course/add-course.component.ts

```diff
@@ -72,9 +72,36 @@ export class AddCourseComponent implements OnInit {
       return;
     }
 
+    this.prereqsList.length = 0;
```

Comment:
is this dup?

Reply:
no clue ¯\_(ツ)_/¯ it was there before

Reply:
I'll remove the duplicate in master.

The reason why we need to do `this.prereqsList.length = 0;` is b/c the component data is persisted. Suppose we add two courses with prereqs and don't have this line. When the second course adds the prereqs to the end of `prereqsList`, this list will already contain the prereqs from the first course, thus resulting in a list with prereqs for both (since it is a global list).

Reply:
Oh true I didn't see it was a duplicate - my bad